### PR TITLE
Implement KCC solution for folder-iam

### PR DESCRIPTION
**Pull request by MingyanZhao**
Hi  @maqiuyujoyce @BrianPi @trodge @AlexBulankou

Please review the PR - implement KCC solution for folder-iam.

Thanks

### CODE REVIEW

File: config-connector/solutions/iam/kpt/folder-iam/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+  Using [kpt](https://googlecontainertools.github.io/kpt/):
+
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/folder-iam folder-iam
+  ```
+
+# REQUIREMENTS
+
+  A working Config Connector cluster using a cnrm-system service account.
```
Comment (trodge) [inline config-connector/solutions/iam/kpt/folder-iam/README.md:22]: Should mention that the service account needs the Security Admin role in the folder.

File: config-connector/solutions/iam/kpt/folder-iam/Kptfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+          value: ${IAM_MEMBER?}
+          setBy: PLACEHOLDER
+      description: GCP identity to grant role to
+    io.k8s.cli.setters.role:
+      x-k8s-cli:
+        setter:
+          name: role
+          value: roles/resourcemanager.folderEditor
+          setBy: package-default
+      description: the IAM role to grant to the member
```
Comment (BrianPi) [inline config-connector/solutions/iam/kpt/folder-iam/Kptfile:29]: Add newline to end of file.

File: config-connector/solutions/iam/kpt/folder-iam/README.md (hunk 1 above)
Comment (MingyanZhao) [reply config-connector/solutions/iam/kpt/folder-iam/README.md]: Added. Folder Admin role is enough to apply the solution so I think we should use that.

File: config-connector/solutions/iam/kpt/folder-iam/Kptfile (hunk 2 above)
Comment (MingyanZhao) [reply config-connector/solutions/iam/kpt/folder-iam/Kptfile]: Done. Thanks!

File: config-connector/solutions/iam/kpt/folder-iam/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+Folder-iam
+==================================================
```
[the remark this answers is not in the document]
Comment (MingyanZhao) [reply config-connector/solutions/iam/kpt/folder-iam/README.md:2]: Done.

File: config-connector/solutions/iam/kpt/folder-iam/README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+  folder-iam
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to grant a specific member a role (default to roles/resourcemanager.folderEditor) to an existing folder.
+
+# CONSUMPTION
+
+  Using [kpt](https://googlecontainertools.github.io/kpt/):
```
[the remark this answers is not in the document]
Comment (MingyanZhao) [reply config-connector/solutions/iam/kpt/folder-iam/README.md:14]: Done.

File: config-connector/solutions/iam/kpt/folder-iam/README.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/folder-iam folder-iam
+  ```
+
+# REQUIREMENTS
+
+  A working Config Connector cluster using a cnrm-system service account with _at least_ the following role in
+  the folder:
```
[the remark this answers is not in the document]
Comment (MingyanZhao) [reply config-connector/solutions/iam/kpt/folder-iam/README.md]: Done

File: config-connector/solutions/iam/kpt/folder-iam/README.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+  Using [kpt](https://googlecontainertools.github.io/kpt/):
+
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/folder-iam folder-iam
+  ```
+
+# REQUIREMENTS
+
+  A working Config Connector cluster using a cnrm-system service account with _at least_ the following role in
```
[the remark this answers is not in the document]
Comment (MingyanZhao) [reply config-connector/solutions/iam/kpt/folder-iam/README.md]: Done

**Pull request merged**